<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>481</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="todohead_hboxlayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="LockBtn">
         <property name="text">
          <string>锁定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TitleLabel">
         <property name="styleSheet">
          <string notr="true">color:white;
font:bold;</string>
         </property>
         <property name="text">
          <string>MyToDo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExitBtn">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="todoitem_vboxlayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="InfoLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="newitem_hboxlayout">
         <item>
          <widget class="QLineEdit" name="NewItemEdit"/>
         </item>
         <item>
          <widget class="QCheckBox" name="newitem_i_checkbox">
           <property name="text">
            <string>要</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="newitem_e_checkBox">
           <property name="text">
            <string>紧</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="ToDoListWidget">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
border:0;</string>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExportToDay">
         <property name="text">
          <string>导出今日工作</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HistoryListSelectLayout">
         <item>
          <widget class="QDateEdit" name="HistoryItemDateEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="HistorySelectBtn">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>选择历史</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="DoneListWidget">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
border:0;</string>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
